specification/sql/resource-manager/Microsoft.Sql/preview/2022-08-01-preview/ReplicationLinks.json

```diff
@@ -307,6 +307,49 @@
     }
   },
   "definitions": {
+    "PartnerDbRg": {
+      "description": "A partner db resource group.",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/PartnerDbRgProperties",
+          "description": "Resource properties.",
+          "x-ms-client-flatten": true
+        },
+        "id": {
+          "description": "Resource ID.",
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "description": "Resource name.",
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "description": "Resource type.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "PartnerDbRgProperties": {
+      "description": "Properties of a partner database resource group.",
+      "type": "object",
+      "properties": {
+        "partnerSubscriptionId": {
+          "format": "uuid",
```

Comment:
guuid lint issue

Reply:
the issue has been resolved in lastest commit